domain.yml

```diff
@@ -211,6 +217,8 @@ responses:
     text: Would you like to schedule a payment of {currency}{amount-of-money}{payment_amount_type}
       towards your {credit_card} account for {time_formatted}?
   utter_ask_transfer_money_form_zz_confirm_form:
+  - text: Would you like to transfer {currency}{amount-of-money} to {PERSON}?
+    channel: twilio_voice
```

Comment:
Does having buttons in a voice channel make sense?

It does not feel natural to me especially if the choice is between "Yes" or "No".

When the bot finished asking "Would you like to transfer ... ?", I can't answer immediately because I need to wait for it to finish uttering each button's text representation i.e. "Yes" and "No, cancel the transactions".